### PR TITLE
add blog post of introdution-of-sxg

### LINKED ISSUE #1903: content: Introduction of SXG.
**Author**: @kumagi
**Reviewer**: TBD

**Target publish date:** 2020-01-20

- [ ] Check this box if this is a hard deadline.
- [ ] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

Check out the [web.dev writing process](https://web.dev/handbook/quick-start/).

When you're ready, make a copy of [this template](https://docs.google.com/document/d/1lgaNIEnXZf-RB8_p9RK22QEgpXJqnu77pLWVWVy4nuw/edit)
and fill in the proposal and outline sections. When you're finished, drop a link
to the doc in the 1st step below and check the box. A member of the web.dev team
will assign this ticket to themselves and work as your reviewer to help you
complete the remaining steps.

- [ ] 1. I've done my proposal https://docs.google.com/document/d/1j5Ib3YuonGkPqMDtIfebnqTnPxhR_bYzwBEal6DDI1U/edit?usp=sharing
- [ ] 2. Proposal approved — _reviewer checks this_
- [ ] 3. I've added a draft to my proposal doc
- [ ] 4. Draft approved — _reviewer checks this_
- [ ] 5. I've submitted a pull request #<add PR number>
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by kumagi**
Fixes #1903

Author: @kumagi
Reviewer: @CrystalOnScript, 

proposal: https://docs.google.com/document/d/1j5Ib3YuonGkPqMDtIfebnqTnPxhR_bYzwBEal6DDI1U/edit?usp=sharing

Changes proposed in this pull request:

- Append contributor information of me.
- Append article of "introduction of sxg".

### CODE REVIEW

**Comment by kaycebasques** [comment]
See https://github.com/GoogleChrome/web.dev/issues/1903#issuecomment-583656074

**Comment by kaycebasques** [comment]
@kumagi please move the content to the Google Doc: https://docs.google.com/document/d/1j5Ib3YuonGkPqMDtIfebnqTnPxhR_bYzwBEal6DDI1U/edit?usp=sharing

It will be faster for me to review and edit it there

Please make sure that I have edit access to the doc

**Comment by kumagi** [comment]
Done. I wrote 2 articles "introduction-of-sxg" and "distributing-sxg".
Both of them are individual branch and the same doc in https://docs.google.com/document/d/1j5Ib3YuonGkPqMDtIfebnqTnPxhR_bYzwBEal6DDI1U/edit#

**Comment by kumagi** [comment]
Can I get review? or any other process is required?

**Comment by kaycebasques** [comment]
@kumagi see https://github.com/GoogleChrome/web.dev/issues/1903#issuecomment-593749737

**Comment by kumagi** [comment]
CI is failed but I cannot reproduce it in my environment with `npm ci`.

**Comment by robdodson** [comment]
The post is missing its yaml front matter.
Take a look at https://github.com/GoogleChrome/web.dev/blob/master/src/site/_drafts/_template-post/index.md for an example of the front matter you need to include.

**Comment by robdodson** [comment]
Looks like the post needs to be rebased or use the resolve conflicts button.

File: src/site/content/en/blog/introduction-of-sxg/index.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
+---
+title: How to setup Signed HTTP Exchanges (SXG)
```
Comment (kaycebasques) [inline src/site/content/en/blog/introduction-of-sxg/index.md]:
```suggestion
layout: post
title: How to setup Signed HTTP Exchanges (SXG)
```

Comment (kaycebasques) [inline src/site/content/en/blog/introduction-of-sxg/index.md]:
```suggestion
title: How to set up Signed HTTP Exchanges (SXG)
```

Comment (kumagi) [reply src/site/content/en/blog/introduction-of-sxg/index.md]: Thanks, Rebased.

**Comment by kaycebasques** [comment]
Hey @kumagi I copied your code to PR #2349 and finished it up and merged the PR. So your post should be live soon.

I did this because I was going to need a lot of small fixes to the markdown and it seemed like it might have been annoying for you to do it and might have taken you a lot of time, whereas it took me less time.

**Comment by kaycebasques** [comment]
@kumagi page is live: https://web.dev/how-to-set-up-signed-http-exchanges/

**Comment by kumagi** [comment]
Amazing!
Thank you very much for your great cooperation!

**Pull request closed without merging**